packages/react/src/UnderlineNav/styles.ts

```diff
@@ -135,17 +137,34 @@ export const menuItemStyles = {
   // To reset the style when the menu items are rendered as react router links
   textDecoration: 'none',
 }
+/**
+ *
+ * @param containerRef The Menu List Container Reference.
+ * @param listRef The Underline Nav Container Reference.
+ * @description This is the styles for our popover menu for the underline nav.
+ */
+export const menuStyles = (containerRef: Element | null, listRef: Element | null): BetterSystemStyleObject => {
+  const minStyleWidth = 192
 
-export const menuStyles = {
-  position: 'absolute',
-  zIndex: 1,
-  top: '90%',
-  right: '0',
-  boxShadow: '0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24)',
-  borderRadius: '12px',
-  backgroundColor: 'canvas.overlay',
-  listStyle: 'none',
-  // Values are from ActionMenu
-  minWidth: '192px',
-  maxWidth: '640px',
+  const baseRules = {
+    position: 'absolute',
+    zIndex: 1,
+    top: '90%',
+    boxShadow: '0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24)',
+    borderRadius: '12px',
+    backgroundColor: 'canvas.overlay',
+    listStyle: 'none',
+    // Values are from ActionMenu
+    minWidth: `${minStyleWidth}px`,
+    maxWidth: '640px',
+  }
```

Comment:
```suggestion
  export const baseRules = {
    position: 'absolute',
    zIndex: 1,
    top: '90%',
    boxShadow: '0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24)',
    borderRadius: '12px',
    backgroundColor: 'canvas.overlay',
    listStyle: 'none',
    // Values are from ActionMenu
    minWidth: `${minStyleWidth}px`,
    maxWidth: '640px',
    right: '0'
  }
```

Reply:
exporting the baseRules with the `right: '0'` for the default styles.

Reply:
I added `right: '0'` back to the `baseMenuStyles`. :100: